src/index.ts

```diff
@@ -2795,6 +3616,119 @@ export class Cinnamon {
         ).data.updateEntitlement;
     }
 
+    @bind
+    async updateEntitlements({
+        input,
+        filter,
+        sort,
+        first,
+        last,
+        after,
+        before,
+        fields = this.defaultEntitlementFields,
+        showDeleted,
+        headers,
+        token,
+    }: {
+        input: EntitlementUpdateInput;
+        filter?: Scalars['FilterInput'];
+        sort?: SortInput;
+        first?: number;
+        last?: number;
+        after?: PageInfo['endCursor'];
+        before?: PageInfo['startCursor'];
+        fields?: EntitlementField[];
+        showDeleted?: boolean;
+        headers?: Headers;
+        token?: string;
+    }) {
+        return (
+            await this.api<'updateEntitlements'>({
+                query: pageMutationGenerator('updateEntitlements', fields, {
+                    input: 'EntitlementUpdateInput!',
+                    showDeleted: 'Boolean',
```

Comment:
:eye::eye: no show deleted for entitlements

Reply:
fixed